en/references/tested-judge/json/README.md

```diff
@@ -0,0 +1,2045 @@
+---
+title: TESTed JSON-testplans
```

Comment:
Another general remark: I feel like this file too detailed. After all, the JSON testplan is aimed at advanced users:

- If needed they can look at the JSON Schema itself, which we can generate from the Python code (if we switch everything to pydantic classes instead of dataclasses, we can even include Python docs I believe).
- They can also look at the testplan Python module (the code there is relatively straightforward)

For example, I don't think it's necessary to describe every enum construct (they are just strings in the json after all).
Additionally, this file right now is a lot of work to update if the testplan format should change.

(Perhaps we'll discuss this in the next meeting)